wiki/Tournaments/OMIC/2019_4K/en.md

```diff
@@ -2,9 +2,415 @@
+| **Grup H** | ::{ flag=ID }:: [\[-TiaZark-\]](https://osu.ppy.sh/users/14294140) (13 Point), ::{ flag=ID }:: [Dil-](https://osu.ppy.sh/users/3796282) (11 Point), ::{ flag=ID }:: [ReZ\_TB](https://osu.ppy.sh/users/9113955) (6 Point), ::{ flag=ID }:: [Hilmi Ghazian](https://osu.ppy.sh/users/3200342) (6 Point), ::{ flag=ID }:: [RizSama](https://osu.ppy.sh/users/9281315) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53009213) |
+
+## General Rules
+
+- There is no limit to the amount of participants.
+- Participant should register on the links that are already provided.
+- Flag on each participant's osu! account must be**Indonesia**.
+- The lowest rank for this tournament is **#5000 Country**.
+- All of the tournament staffs are not permitted to participate as a player on this tournament, except for staff that are taking the position of the following:
+  - Streamer 
+  - Commentator 
+  - Designer 
+- Every participant who registers will go through a session of [Tournament Screening](/wiki/Tournaments/Official_support#tournament-screening).
+
+### Tournament Rules
+
+- Matches will be held every Saturday and Sunday.
+- The scoring system for this tournament is Score V2.
+- Mappool for each round will be **announced by the staff** every Sunday after the previous round is over.
+- All players should **download all maps** for the mappool before the match starts.
+- If there is no referee during the match time, please inform the **Organizer staff**.
+- Participants are expected to **show up on time** according to the schedule.
+- The staff can only give a delay of **10 minutes**.
+- If a player has a connection problem that they left the match, then the decision will be **determined by the referee** which is **absolute**.
+- Players must follow the [rules](/wiki/Rules) that has been set by the game.
+
+### Match Regulations 
+
+- All players who follow and fulfill all the registration requirements of this tournament will enter the Qualifiers round.
+- Each lobby contains a maximum of 8 players in the Qualifiers round.
+- Players are allowes to use visual mods such as (HD), Flashlight (FL), Fade In (FI), and Mirror (MR).
+- Players will play 2 maps for a warm-up session that had been selected by the player that rolled the highest.
+- **Babak Kualifikasi**
+- The maximum number of players to enter the next round is 56 players.
+- In the Qualifiers round there will be 4 maps in the mappool that have different skillsets, all of these maps will be created by the mappool selector.
+- All players will have 2 chances to play in each map, which means a total of 8 songs played.
+- The highest score played for each beatmap will be accounted and the Qualifiers score will be accumulated based on the comparison of scores of other players.
+- After the Qualifiers round ends there will be a distribution of players based on seed.
+  - The function of seed is for the distribution of the next round, where players are randomized based on different seed.
```

Comment:
is it randomized if the seed is determined based on their position in the qualifiers?

Reply:
correct

Reply:
...then it's not randomized. it can't be because it's not random.

i'm also pretty sure this whole sentence is redundant, looking at the above and below points